packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -124,163 +151,7 @@ export function getValidatorApi({
     return genesisBlockRoot || ZERO_HASH;
   }
 
-  /**
-   * If advancing the local clock `MAX_API_CLOCK_DISPARITY_MS` ticks to the requested slot, wait for its start
```

Comment:
are these comments no longer valid? please keep

Reply:
No comments are completely removed, either updated or moved to specific utility functions.